src/index.html

```diff
@@ -195,6 +197,138 @@ <h2 class="newjob__title">新着求人</h2>
         </div>
       </div>
     </div>
+
+    <!-- ギズジョブの4つの特徴 -->
+    <section class="p-contents l-inner">
+      <h2 class="p-contents__title">ギズジョブの4つの特徴</h2>
+      <ul class="p-contents__list">
+        <li class="p-contents__list__item">
+          <h3 class="item__title">WEB上で応募完結</h3>
+          <p class="item__text">エントリーシートの記入を済ませれば、あとは気になる企業に応募して<br>待つだけで面接アポまで完結！</p>
```

Comment:
[must]
`p-contents`の中のElementなので、以下のように先頭に`p-contents`をつける必要があります！
FLOCSSでスタイルを当てる時は、○-の接頭辞がついていない時点で命名規則に違反している可能性を疑った方が良いと思います！

```suggestion
          <h3 class="p-contents__item__title">WEB上で応募完結</h3>
          <p class="p-contents__item__text">エントリーシートの記入を済ませれば、あとは気になる企業に応募して<br>待つだけで面接アポまで完結！</p>
```

Reply:
プレフィックスできていなかったところ確認して、修正しました！
521039e11b7f10e91afbbf2924d537bb50439508

次回から命名する際は命名規則に違反していないか確認するようにします。